chain/bitcoind.go

```diff
@@ -916,8 +936,13 @@ func (c *BitcoindClient) filterBlock(block *wire.MsgBlock, height int32,
 		return nil, nil
 	}
 
-	log.Debugf("Filtering block %d (%s) with %d transactions", height,
-		block.BlockHash(), len(block.Transactions))
+	// Only mention that we're filtering a block if the client wallet has
+	// started monitoring the chain.
+	if !c.notifying() {
+		log.Debugf("Filtering block %d (%s) with %d transactions",
+			height, block.BlockHash(), len(block.Transactions))
+	}
+
```

Comment:
Ah or seems fine as it's set here below?

Reply:
Yes, but I've gone ahead and added that above as well.